play/src/front/Chat/ChatSidebar.svelte

```diff
@@ -25,6 +25,14 @@
     const isUserListEnabled = gameScene.room.isChatOnlineListEnabled;
 
     function onKeyDown(e: KeyboardEvent) {
+        const activeElement = document.activeElement as HTMLElement | null;
+        const dataTestId: string | null | undefined = activeElement?.getAttribute("data-testid");
```

Comment:
`data-testid` is used especially to playright testing. `activeElement.tagName === "INPUT"` must be sufficient?